apsuite/optics_analysis/coupling_correction.py

```diff
@@ -105,56 +108,58 @@ def set_ksl(self, model=None, skewidx=None, ksl=None):
         return newmod
 
     @staticmethod
-    def get_fm(res):
+    def get_figm(res):
         """Calculate figure of merit from residue vector."""
         return np.sqrt(np.sum(np.abs(res)**2)/res.size)
```

Comment:
Why did you have to take the `np.abs` of `res` here?
I think that for long arrays `res*res` is faster than `res**2`.

Reply:
my bad!